src/js/math.ts

```diff
@@ -4,6 +4,29 @@ import _ from "underscore";
 
 import { tolerance as TOLERANCE } from "./constants";
 
+export type MathArray = mathjs.MathArray;
```

Comment:
Let's explain why this is necessary in a comment and put a TODO to re-export all types when updating Typescript version

Reply:
Done